src/extensions/default/MDNDocs/main.js

```diff
@@ -102,6 +103,15 @@ define(function (require, exports, module) {
             return null;
         }
 
+        // Send analytics data for Quick Doc open
+        HealthLogger.sendAnalyticsData(
+            "cssQuickDoc",
+            "usage",
```

Comment:
2nd param is event category? usage might not be the correct word. feature name should be there something like quickedit, multicursor, refactor etc

Reply:
@SnchitGrover would be able to clarify this.